src/netex_idf/common.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017-2019 Kisio Digital and/or its affiliates.
+//
+// This program is free software: you can redistribute it and/or
+// modify it under the terms of the GNU General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+use super::offers::{self, GeneralFrameType, COMMON_FILENAME, NETEX_COMMON};
+use crate::{
+    minidom_utils::{TryAttribute, TryOnlyChild},
+    netex_utils::{self, FrameType},
+    objects::{Comment, CommentType},
+    Result,
+};
+use failure::format_err;
+use minidom::Element;
+use transit_model_collection::CollectionWithId;
+
+pub fn parse_common(common: &Element) -> Result<CollectionWithId<Comment>> {
+    let frames = netex_utils::parse_frames_by_type(common.try_only_child("dataObjects")?)?;
+    let general_frames = frames
+        .get(&FrameType::General)
+        .ok_or_else(|| format_err!("Failed to find a GeneralFrame in {}", COMMON_FILENAME))?;
+    let general_frames_by_type = offers::parse_general_frame_by_type(general_frames)?;
+    let common_general_frame = general_frames_by_type
+        .get(&GeneralFrameType::Common)
+        .ok_or_else(|| format_err!("Failed to find the GeneralFrame of type {}", NETEX_COMMON))?;
+    let comments = common_general_frame
+        .only_child("members")
+        .iter()
+        .flat_map(|members| members.children())
+        .filter_map(|notice_element| {
+            println!("BEFORE");
```

Comment:
why not but no :)
same for all the println below

Reply:
Oops...

Reply:
Fixed, and I also added one missing line which was merging the list of comments into the `Collections`.